executor/adapter.go

```diff
@@ -118,9 +119,6 @@ func (a *recordSet) NewChunk() *chunk.Chunk {
 func (a *recordSet) Close() error {
 	err := a.executor.Close()
 	a.stmt.logSlowQuery(a.txnStartTS, a.lastErr == nil)
-	if a.processinfo != nil {
```

Comment:
Is the `processinfo` used?

Reply:
Is the two same?

Reply:
Just by mistake, `pi` is not used in this function, so i delete it.